group_vars/all/osa.yml

```diff
@@ -115,20 +115,31 @@ glance_glance_api_conf_overrides:
   DEFAULT:
     image_cache_dir: None
 
+# Elasticsearch global variables
+elasticsearch_http_port: 9200
+elasticsearch_tcp_port: 9300
+
 # Adding elastic search and kibana to haproxy_extra_services
 haproxy_extra_services:
   - service:
-      haproxy_service_name: elasticsearch
-      haproxy_backend_nodes: "{{ groups['elasticsearch'] | default([]) }}"
+      haproxy_service_name: elasticsearch-http
```

Comment:
Does this *need* the name change. I think this might make upgrades a little more complicated than it needs to be.